### PR TITLE
Updated requirements to ranges

### PR DESCRIPTION
**Pull request by clintonb**
If we are going to read setup.py requirements from files, the files need to use ranges. Otherwise, we will end up with requirement conflicts when the main project wants to upgrade requirements.

### CODE REVIEW

**Comment by clintonb** [comment]
This is necessary to support upgrading Django REST Framework for edx-platform. See https://github.com/edx/edx-platform/pull/13945.

**Comment by clintonb** [comment]
@cahrens @efischer19 please review

File: requirements/test.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+edx-lint>=0.5.1,<1.0.0
+freezegun>=0.1.11,1.0.0
+mock>=1.0.1,<2.0.0
+moto>=0.3.1,<1.0.0
+pep8>=1.7.0,<2.0.0
+factory_boy>=2.6.1,3.0.0
 
 git+https://github.com/edx/django-pyfs.git@1.0.3#egg=django-pyfs==1.0.3
-git+https://github.com/edx/i18n-tools.git@56f048af9b6868613c14aeae760548834c495011#egg=i18n_tools
-git+https://github.com/edx/edx-lint.git@v0.4.1#egg=edx_lint==0.4.1
```
Comment (efischer19) [inline requirements/test.txt]: Why were these 2 removed? I know edx/i18n-tools is needed for translation updates at least. It probably should be updated to a more recent version like [v0.3.4](https://github.com/edx/i18n-tools/releases/tag/v0.3.4), and may not be in the right location, but removing it entirely seems like a problem. edx-lint I'm less concerned about.

File: .travis.yml [hunk 2]
```diff
@@ -34,9 +34,5 @@ before_script:
 script:
   - "make test"
   - "python manage.py makemessages -l eo"
-branches:
-    only:
-      - master
-      - ora-staff-grading
```
Comment (efischer19) [inline .travis.yml]: For my own edification - what's the impact of this change? Just that Travis builds will run more regularly on non-master branches?

File: setup.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='1.1.10',
+    version='1.1.11',
```
Comment (efischer19) [inline setup.py]: Please leave this at the current version for this PR, and I'll take care of updating it with translations as part of the [ORA release process](https://openedx.atlassian.net/wiki/display/ENG/ORA2+Release+Process). @cahrens, do you or your team have any work you'd like to release soon, or should I get this process started once this PR lands?

Comment (cahrens) [inline setup.py]: @efischer19 go ahead and start the release process-- I did merge something on Friday, so I will have something to verify.

File: .travis.yml (hunk 2 above)
Comment (clintonb) [inline .travis.yml]: Correct. See https://docs.travis-ci.com/user/customizing-the-build/#Building-Specific-Branches.

**Comment by cahrens** [comment]
@clintonb Did you verify that the a11y and acceptance tests pass with these changes? Note that you will have to create a sandbox and run them manually, since they don't run as part of the travis build.

File: requirements/test.txt (hunk 1 above)
Comment (clintonb) [inline requirements/test.txt]: Both of these are hosted on PyPI. I added them to the list above.

File: setup.py (hunk 3 above)
Comment (clintonb) [inline setup.py]: Okay. Reverting.

**Comment by clintonb** [comment]
@cahrens I have not run those tests. Time permitting, I'll run them today and post back here.

**Comment by clintonb** [comment]
Have you given any thought to integrating those tests here into this repo?

File: .travis.yml (hunk 2 above)
Comment (efischer19) [inline .travis.yml]: Cool, thanks!

File: requirements/test.txt (hunk 1 above)
Comment (efischer19) [inline requirements/test.txt]: Ah, I had missed that initially. Great!

**Comment by cahrens** [comment]
@clintonb The integration/a11y tests are in this repo. They just run through Jenkins jobs on master, instead of as part of the travis build.

See https://openedx.atlassian.net/wiki/display/TNL/ORA+FAQ#ORAFAQ-Runningacceptancetests

**Comment by efischer19** [comment]
Implemented in https://github.com/edx/edx-ora2/pull/1023, containing the latest DRF updates as certified by the Platform team

**Pull request closed without merging**